Builds/ArchLinux/PKGBUILD

```diff
@@ -25,9 +25,6 @@ build() {
 }
 
 check() {
-	cd "$srcdir/$pkgname"
```

Comment:
seems likely that it would work without the chdir...@wilsonianb, how can I vet changes to PKGBUILD ?